languages/clojure/_sidebar.md

```diff
@@ -0,0 +1,11 @@
+[Start here](/)
```

Comment:
is this set as link necessary? would be confusing as it brings out 404 if clicked. I see c# is the same, might be reserved for later targeting?

Reply:
I just built the site locally (using these [instructions](https://github.com/exercism/v3/blob/master/docs/maintainers/website.md)) and it seems to be doing the right thing.